_data/nav.yml

```diff
@@ -57,6 +57,13 @@
       page: /resources/dev/
       tpl: dev.html
 
+    - id: resources.ml
+      i18n: resources.ml
+      name: ML
+      ref: resources.dev
```

Comment:
shouldn't this be `resources.ml`? :thinking:

Reply:
ohh hindsight will correct it

Reply:
@king-11 updates?

Reply:
@nishantkr18  updated